lib/headquarters/client.rb

```diff
@@ -0,0 +1,13 @@
+require 'headquarters/api'
+
+module Headquarters
+  class Client < API
+    namespace :members
```

Comment:
It would actually have to be 

```ruby
def members
  @members ||= Members.new(authenticator: authenticator)
end
```

so the memoization and the authenticator argument would end up being duplicated in all of those methods. Plus, `namespace :members` sounds looks more semantically correct IMO.

a method called members might imply that it returns the list of members, and not a client (this from the perspective of someone reading the code, not a user of the gem)